test_files/async_functions/async_functions.js

```diff
@@ -1,71 +1,50 @@
-/**
- * @fileoverview added by tsickle
- * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
- */
 goog.module('test_files.async_functions.async_functions');
 var module = module || { id: 'test_files/async_functions/async_functions.ts' };
 module = module;
 exports = {};
 const tslib_1 = goog.require('tslib');
+/**
```

Comment:
hu, why does this end up below the `goog.module` etc? That seems like a bug.

Can you figure out why this happens? What's different here?

Reply:
It appears to be something going wrong when it's a @fileoverview comment.

Since it's not relevant to this PR, I undid that part of it.

Reply:
I don't see the change yet?

Reply:
The @fileoverview is still at the top of hte file, unmodified.  This is just a regular comment.

Reply:
Ping, don't forget me!

Reply:
Wait, you approved and I just missed it, ha!